src/main/java/stelle/task/Event.java

```diff
@@ -0,0 +1,67 @@
+package stelle.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import stelle.exception.WrongDateFormatException;
+
+/**
+ * This class represents an stelle.task.Event. It is child of the stelle.task.Task class.
+ * @author Lee Ze Hao (A0276123J)
+ */
+
+public class Event extends Task {
+    private static final String DATE_TIME_INPUT_FORMATTER_PATTERN = "yyyy-MM-dd HH:mm";
+    private static final String DATE_TIME_OUTPUT_FORMATTER_PATTERN = "d LLLL yyyy HH:mm";
+    private static final DateTimeFormatter dateTimeInputFormatter = DateTimeFormatter
+            .ofPattern(DATE_TIME_INPUT_FORMATTER_PATTERN);
+    private static final DateTimeFormatter dateTimeOutputFormatter = DateTimeFormatter
+            .ofPattern(DATE_TIME_OUTPUT_FORMATTER_PATTERN);
+    private LocalDateTime from;
+    private LocalDateTime to;
+
+    /**
+     * Constructor for an stelle.task.Event object. Sets name upon creation.
+     * @param name Name of stelle.task.Event object.
+     * @param from A string indicating starting date and time of event.
+     * @param to A string indicating end date and time of event.
+     */
+    public Event(String name, String from, String to) {
+        super(name, TaskType.EVENT);
+        try {
+            this.from = LocalDateTime.parse(from.strip(), dateTimeInputFormatter);
+            this.to = LocalDateTime.parse(to.strip(), dateTimeInputFormatter);
+        } catch (Exception e) {
+            throw new WrongDateFormatException();
```

Comment:
Should the date format validation be done outside of the `Event` class, e.g. in the `Parser` class instead? To me, this feels more appropriate since the `Parser` should handle all input-related matters. Additionally, validating the dates in the `Parser` saves you from rewriting a few lines of code in the other `Deadline` class.